sound/soc/sof/intel/hda-loader.c

```diff
@@ -310,18 +305,6 @@ int hda_cl_copy_fw(struct snd_sof_dev *sdev, struct hdac_ext_stream *hext_stream
 		return ret;
 	}
 
-	if (sdev->pdata->ipc_type == SOF_IPC_TYPE_4) {
-		/* Wait for completion of transfer */
```

Comment:
this is not going to apply to the commit pointed by Fixes.
I think you should point to the next patch which excluded IPC3, that one is fixes the original and this will follow it.

Reply:
Ack @ujfalusi , you are right. Better to point to the newer patch, updated in V3.